ci/pipelines/e2e-helpers.yml

```diff
@@ -0,0 +1,105 @@
+---
+resources:
+  - name: reverse-proxy-src
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/alphagov/pay-scripts
+      branch: master
+      username: alphagov-pay-ci
+      password: ((github-access-token))
+      paths:
+        # Should trigger on any changes to this folder
+        - images/proxy/*
```

Comment:
These are glob patters, are you happy for it only to be changes in images/proxy or do you need subfolders too (in which case you want `**` I believe. You might need **/* 🤔

Reply:
Just that (flat) folder, think we're ok https://github.com/alphagov/pay-scripts/tree/master/images/proxy

Reply:
Cool 👍